page_data.go

```diff
@@ -4,90 +4,120 @@ import "time"
 
 const pageDataURL = "/w/api.php"
 
+// PageDataOresScore representation for ORES score
+type PageDataOresScore struct {
+	True  float64 `json:"true"`
```

Comment:
Seems a bit unintuitive : True / False of type float.
Are these TrueProbability and FalseProbability ?

Reply:
Yeah, but this is how actions API returns them.